main.tf

```diff
@@ -60,11 +60,16 @@ resource "aws_elasticache_replication_group" "default" {
   parameter_group_name          = "${aws_elasticache_parameter_group.default.name}"
   availability_zones            = ["${slice(var.availability_zones, 0, var.cluster_size)}"]
   automatic_failover_enabled    = "${var.automatic_failover}"
+  apply_immediately             = "${var.apply_immediately}"
```

Comment:
didn't found apply_immediately in the variable file

Reply:
Lines 133-136, was already present but not used.